docs/source/models.rst

```diff
@@ -232,3 +232,8 @@ Weighted Matrix Factorization (WMF)
 --------------------------------------------------
 .. automodule:: cornac.models.wmf.recom_wmf
    :members:
+
+Embarrassingly Shallow Autoencoders for Sparse Data (EASEᴿ)
```

Comment:
This should be added after the AMR model. The list is organized chronologically.